### PR TITLE
Update WordPress tutorial

### LINKED ISSUE #1138: Update the WordPress tutorial
* Update the code-blocks that are "bash" to "none" for mariaDB CLIs.  

* Reference * 
https://docs.01.org/clearlinux/latest/tutorials/wordpress/web-server-install.html#web-server-install

### PR DESCRIPTION
**Pull request by bktan8**
Closes #1138

Signed-off-by: Bun K Tan <bun.k.tan@intel.com>

### CODE REVIEW

File: source/tutorials/lamp-server.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. _web-server-install:
+.. _lamp-server-install:
```
Comment (mvincerx) [inline source/tutorials/lamp-server.rst:1]: We'd like to preserver the URL. Doc title can change though.

Comment (bktan8) [reply source/tutorials/lamp-server.rst:1]: my reason for changing the URL is that "web-server-install" is too generic.  It could be Apache, nginx, etc.

Comment (mvincerx) [reply source/tutorials/lamp-server.rst:1]: @krisdale.  Please take a quick look to confirm that this doc has low hits. 
Please submit a ticket. Thanks.

Comment (mvincerx) [reply source/tutorials/lamp-server.rst:1]: Confirmed: hit rate is low. Keep as is, @bktan8.

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**